build/tools/build-debug.py

```diff
@@ -73,6 +78,9 @@ def main():
             if not is_elf(filename):
                 continue
 
+            if sh("readelf -S %s | grep -q '.debug_info'" % (filename), nofail=True) != 0:
+                continue
```

Comment:
If we are using readelf here should we remove is_elf() check above?

Reply:
As I see is_elf() is much more trivial, just checking signature at the beginning of the file.  It may be faster then run couple binaries.